db/migrations/20200915204154_create_flip_file_cat.sql

```diff
@@ -0,0 +1,23 @@
+-- +goose Up
+CREATE TABLE maker.flip_file_cat
+(
+    id         SERIAL PRIMARY KEY,
+    header_id  INTEGER NOT NULL REFERENCES public.headers (id) ON DELETE CASCADE,
+    log_id     BIGINT  NOT NULL REFERENCES public.event_logs (id) ON DELETE CASCADE,
+    address_id INTEGER NOT NULL REFERENCES public.addresses (id) ON DELETE CASCADE,
+    msg_sender INTEGER NOT NULL REFERENCES public.addresses (id) ON DELETE CASCADE,
+    what       TEXT,
+    data       TEXT,
```

Comment:
Not necessarily required, but I wonder if we could make this column a FK to the address table?